server/node-postgres-mydb/db/seeds/dev/01_song.js

```diff
@@ -0,0 +1,31 @@
+
+const records = require(' ../../../seed.json');
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('song').del()
+    .then(function () {
+      const songsList= [];
+      records.map(record =>{
+        songsList.push({
```

Comment:
Remember the pupose of this seed file is to _seed the songs_.

Songs are stored in an array in record.tracks.all, so you need to do a map or a forEach over this array to seed all of the songs indiviually.

Remember that the song DB table contains some data that reside under "record" in the JSON, so you have to do some re-mapping for each song.